slack_bolt/app/async_app.py

```diff
@@ -282,17 +287,36 @@ def listener_runner(self) -> AsyncioListenerRunner:
     # -------------------------
     # standalone server
 
+    from .async_server import AsyncSlackAppServer
+
+    def server(
+        self, port: int = 3000, path: str = "/slack/events"
+    ) -> AsyncSlackAppServer:
+        """Configure a web server using AIOHTTP.
+
+        :param port: The port to listen on (Default: 3000)
+        :param path: The path to handle request from Slack (Default: /slack/events)
+        :return: None
+        """
+        if (
+            self._server is None
+            or self._server.port != port
+            or self._server.path != path
+        ):
+            self._server = AsyncSlackAppServer(port=port, path=path, app=self,)
+        return self._server
+
+    def web_app(self, path: str = "/slack/events") -> web.Application:
```

Comment:
When you use aiohttp-devtools, specifying the port to listen on in code does not work. That's why I removed the arg in this factory method as it's so confusing if giving it here does not work.